### PR TITLE
Added markers corresponding to each post that has a location, #19

### PR DESCRIPTION
**Pull request by itumejia**

### CODE REVIEW

File: app/src/main/java/com/example/dogbook/main/fragments/MapFragment.java [hunk 1]
```diff
@@ -132,4 +155,39 @@ private void displayLocation() {
         CameraUpdate cameraUpdate = CameraUpdateFactory.newLatLngZoom(latLng, MAP_FOCUS_ZOOM);
         map.animateCamera(cameraUpdate);
     }
+
+    private void getLocationPosts() {
+        ParseQuery<Post> query = ParseQuery.getQuery(Post.class);
+        query.orderByDescending("createdAt");
+        query.include("author");
+        query.whereExists("location");
```
Comment (nicholeic) [inline app/src/main/java/com/example/dogbook/main/fragments/MapFragment.java:163]: Perhaps we move this out into a class that handles creating this fetch like we talked about before?  So, you can do something like:

```
ParseDataQuery.getLocationPostQuery()
```

File: app/src/main/java/com/example/dogbook/main/fragments/MapFragment.java [hunk 2]
```diff
@@ -132,4 +155,39 @@ private void displayLocation() {
...
+                    return;
+                }
+                Log.e(TAG, "Issue finding posts in Parse", e);
+                Toast.makeText(getContext(), "Unable to refresh posts", Toast.LENGTH_SHORT).show();
+            }
+        });
+    }
+
+    private void addMarkers() {
+        for (Post post: posts) {
```
Comment (nicholeic) [inline app/src/main/java/com/example/dogbook/main/fragments/MapFragment.java:180]: nit: spacing for the semi colon

**Review by nicholeic — COMMENTED** [review]
Would be good to add an image or video of this.  Otherwise, looks good!

**Comment by itumejia** [comment]
> Would be good to add an image or video of this. Otherwise, looks good!

I was just recording that :D 
Kapture 2021-07-20 at 18 02 20

**Pull request merged**